src/lib/mocking/openApiRulesEngine.js

```diff
@@ -372,16 +368,18 @@ const replaceVariablesFromRequest = (inputObject, context, req) => {
   const environment = objectStore.get('inboundEnvironment')
   const matchedArray = resultObject.match(/{\$([^}]+)}/g)
   if (matchedArray) {
-    matchedArray.forEach(element => {
-      // Check for the function type of param, if its function we need to call a function in custom-functions and replace the returned value
+    const userConfig = await Config.getUserConfig(req.customInfo.sourceUser)
+    for (const index in matchedArray) {
+      const element = matchedArray[index]
```

Comment:
I think better to use for/of instead of for/in? Avoid using for/in over an array unless you're certain you mean to iterate over non-numeric keys and inherited keys.